.php_cs.dist

```diff
@@ -9,6 +9,7 @@ $finder = PhpCsFixer\Finder::create()
 ;
 
 return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
```

Comment:
This configuration is actually managed by [phpactor-hub](https://github.com/phpactor/phpactor-hub) and needs to be updated. Just updating here so `composer integrate` works as expected.